### DISCUSSION TITLE
Streaming is hanging

### DISCUSSION THREAD

**Discussion by jerodvenemafm** [deepgram/community#610] [General help] [answered]
### Which Deepgram product are you using?

Deepgram Console

### Details

Calls to the streaming api are hanging without producing any transcript. The "upload" version works fine, but all calls to the live system are just hanging as of about 2pm PST.

<img width="1608" alt="image" src="https://github.com/deepgram/community/assets/65055355/e8bf27f2-02e4-4a35-b82e-b293b2ec9f8b">

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

 /v1/listen?encoding=linear16&punctuate=true&sample_rate=48000&model=nova-2-general

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

6c215acc-300e-45d6-8d79-1d9e74483701

### If possible, please attach your code or paste it into the text box.

The code works, this is in production and was working until 2pm or so today.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@jerodvenemafm  it's possible you tried this during a period of time where our Deepgram Streaming API was having issues.  Are you still having this issue?

You can see our Status History here for reference: https://status.deepgram.com/
